lib/posix/posix_utils.nim

```diff
@@ -127,3 +127,21 @@ proc osReleaseFile*(): Config {.since: (1, 5).} =
     if fileExists(osReleaseFile):
       return loadConfig(osReleaseFile)
   raise newException(IOError, "File not found: /etc/os-release, /usr/lib/os-release")
+
```

Comment:
> # Where possible, contribute OS-independent procs in `os <os.html>`_ instead.

maybe this should be in os.nim?

(let's see what other people think first)

Reply:
I agree. Unfortunately, I didn't have Windows installed, but I've rented a VM with Windows Server on Google Cloud Platform for that purpose. I did the necessary changes. Let's wait for CI and other people's opinions.